app/src/main/java/org/wikipedia/yearinreview/YearInReviewSlides.kt

```diff
@@ -288,13 +262,30 @@ class YearInReviewSlides(
     }
 
     private fun unlockedIconRoute(): List<YearInReviewScreenData> {
+        val isIconUnlocked = yearInReviewModel.userEditsCount > 0 || Prefs.donationResults.isNotEmpty()
+        val isNextYear = LocalDate.now().year > 2025
         return if (isIconUnlocked) {
             listOf(
-                newIconUnlockedScreen()
+                YearInReviewScreenData.CustomIconScreen(
+                    headlineText = R.string.year_in_review_app_icon_title_unlocked,
+                    bodyText = R.string.year_in_review_app_icon_body_unlocked,
+                )
+            )
+        } else if (isNextYear) {
+            listOf(
+                YearInReviewScreenData.CustomIconScreen(
+                    headlineText = R.string.year_in_review_app_icon_title_unlock_later,
+                    bodyText = R.string.year_in_review_app_icon_body_unlock_later,
+                    showDonateButton = true
```

Comment:
If the `showDonateButton` is `true`, the donate button on the toolbar should be hidden.

Reply:
done!